templates/components/quantity.md

```diff
@@ -1,6 +1,4 @@
 {% if value.quantity.unit.is_equivalent('mol / m^3') %}
   {{- '{0:.3g}'.format(value.quantity.to('mol / l').value|float) }} M
 {%- else %}
-  {{- value.quantity.value }} {{ value.quantity.unit.to_string("latex_inline") -}}
-{% endif %}
-
+$`\mathsf{ {{- value.quantity.value }} \, {{ value.quantity.unit.to_string("latex_inline")[8:-1] -}}}`${% endif %}
```

Comment:
```suggestion
  {{- value.quantity.value }} $`{{ value.quantity.unit.to_string("latex_inline").replace("mathrm", "mathsf")[1:-1] -}}`$
{% endif %}
```
I think there's not reason to typeset the value in math mode, is there?

I haven't tried this but this seems a bit less cryptic to me than the `[8:-1]` you had before.

Reply:
Thanks for the suggestion. I think there is only the benefit to control the distance in math mode. I think this is not possible when not in math mode, is it?

Reply:
It's not possible to control spacing when not in LaTeX, at least not easily.

Reply:
How important is fine control of the spacing for us?